### PR TITLE
Update version 23608

### PR DESCRIPTION
**Pull request by roncain**

### CODE REVIEW

**Comment by roncain** [comment]
Reviewers: this will not pass OuterLoops entirely until it is rebased onto PR #599.  Moving to the newer packages brings in the System.Net.Security change that throws when the SSL3 protocol is used.

File: src/Common/test-runtime/project.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
     "frameworks": {
        "dnxcore50": { 
          "dependencies": {
-           "System.Net.Http.WinHttpHandler": "4.0.0-rc2-23602",
+           "System.Net.Http.WinHttpHandler": "4.0.0-rc2-23608",
            "System.Net.Security": "4.0.0-rc2-*"
```
Comment (iamjasonp) [inline src/Common/test-runtime/project.json]: Crap, I missed the System.Net.Security change last time. Can you update this please?

**Comment by iamjasonp** [comment]
We should check whether or not `"xunit.netcore.extensions": "1.0.0-prerelease-*"` should be updated in the same way. 

The latest I see is `xunit.netcore.extensions 1.0.0-prerelease-00131` from [here](https://www.myget.org/feed/dotnet-buildtools/package/nuget/xunit.netcore.extensions)

File: src/System.Private.ServiceModel/tests/Unit/project.json [hunk 2]
```diff
@@ -1,57 +1,57 @@
...
+    "System.Xml.XmlSerializer": "4.0.11-rc2-23608",
     "xunit": "2.1.0",
     "xunit.netcore.extensions": "1.0.0-prerelease-*"
   },
   "frameworks": {
     "dnxcore50": {
       "dependencies": {
-        "System.Net.NameResolution": "4.0.0-rc2-23602",
+        "System.Net.NameResolution": "4.0.0-rc2-23608",
         "System.Net.Security": "4.0.0-rc2-*",
```
Comment (iamjasonp) [inline src/System.Private.ServiceModel/tests/Unit/project.json]: Ditto for S.Net.Security here too, please update from *

**Comment by iamjasonp** [comment]
Synced to TFS change 1556232, applied changes in this PR and built - build passed.

**Comment by iamjasonp** [comment]
LGTM subject to addressing the comments above

**Comment by roncain** [comment]
I would like to address xunit.netcore.extensions in a separate PR.  I tried but experienced issues resolving it, possibly related to build tools.  And this PR needs to go in today.

**Comment by iamjasonp** [comment]
Sounds good without xunit.netcore.extensions. Merge at will.

**Comment by roncain** [comment]
This build successfully under TFS for me too, so I will merge and track the rolling builds

**Pull request merged**